Content.Client/Doors/AirlockVisualizer.cs

```diff
@@ -192,11 +199,25 @@ public override void OnChangeData(AppearanceComponent component)
                 boltedVisible = true;
             }
 
+            if (component.TryGetData(DoorVisuals.EmergencyLights, out bool eaLights) && eaLights)
+            {
+                emergecyLightsVisible = true;
+            }
+
             if (!_simpleVisuals)
             {
                 sprite.LayerSetVisible(DoorVisualLayers.BaseUnlit, unlitVisible && state != DoorState.Closed && state != DoorState.Welded);
                 sprite.LayerSetVisible(DoorVisualLayers.BaseWelded, weldedVisible);
                 sprite.LayerSetVisible(DoorVisualLayers.BaseBolted, unlitVisible && boltedVisible);
+                if (_emergencyAccessLayer)
+                {
+                    sprite.LayerSetVisible(DoorVisualLayers.BaseEAccess, 
```

Comment:
Just call the enum BaseEmergencyAccess because the enum already has to be enum.DoorVisualLayers.BaseEAccess and this makes it clearer.

Reply:
Done.